Move Run A Testsuite on Device Tutorial to VEE Port Guide (#986)
MicroEJ/docs: VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst

.. _run_fs_test_suite_on_esp32_wrover:

> Run FS Test Suite on ESP32-WROVER VEE Port

Reviewer: ```suggestion
Run the FS Test Suite on ESP32-WROVER VEE Port
```

==========================================

This guide describes all the steps to configure and run a
:ref:`vee_port_testsuite` on a device using the `Port
Qualification Tools
<https://github.com/MicroEJ/VEEPortQualificationTools>`_.

The target device is the Espressif ESP32-WROVER-KIT
V4.1 board and the Filesystem Test Suite for :ref:`FS <pack_fs>`
module will be used as an example.

This guide should take 1 hour to complete (excluding the VEE Port

> Getting Started setup).

Reviewer: ```suggestion
setup fron the Getting Started page).
```

Author: Fixed

Prerequisites
-------------

This guide assumes the following:

- Good knowledge of the :ref:`chapter-glossary`.

- Tutorial :ref:`tutorialUnderstandMicroEJFirmwareBuild` has been followed.

- MICROEJ SDK distribution 20.07 or more (see :ref:`get_sdk_version`).

- The `WROVER VEE Port
  <https://github.com/MicroEJ/Platform-Espressif-ESP-WROVER-KIT-V4.1/tree/1.6.2>`__
  has been properly setup (i.e., it can be used to generate a Mono-Sandbox Executable).

> The explanation can be adapted to run the test suite on any other VEE Port providing:

Reviewer: ```suggestion
The instructions of this page can be adapted to run the test suite on any other VEE Port providing:
```
